openstudiocore/src/model/Building.cpp

```diff
@@ -126,6 +126,57 @@ namespace detail {
     SpaceVector spaces = this->spaces();
     result.insert(result.end(), spaces.begin(), spaces.end());
 
+    // TODO: JM 2019-05-13: handle HVAC (#2449)
```

Comment:
@jmarrec is there an issue for this todo?  Also, how does this compare to clone?  I thought building.clone handled airloops?

Reply:
No it's a Todo for https://github.com/NREL/OpenStudio/issues/2449#issuecomment-491901154